vendor/jquery/component.dollar.js

```diff
@@ -1,12 +1,14 @@
-Ember.Component.reopen({
-  $(sel) {
-    Ember.assert(
-      "You cannot access this.$() on a component with `tagName: ''` specified.",
-      this.tagName !== ''
-    );
+(function() {
+  Ember.Component.reopen({
+    $(sel) {
+      Ember.assert(
```

Comment:
Now that I think about it, I think we do need to use:

```js
import { assert } from '@ember/debug';
```

to get the proper stripping...

IMHO, this is safe (even when we move Ember to real modules, we would still handle this specially).